dotCMS/src/test/java/com/dotmarketing/db/DataSourceStrategyProviderTest.java

```diff
@@ -27,18 +27,21 @@
                 "DBProperties",
                 "SystemEnv",
                 "DockerSecret",
-                "Tomcat"
+                "Tomcat",
+                "CustomProvider"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "CustomProvider" appears 4 times in this file; the first occurrence is on line 31](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5092420)